v2/kafka-to-bigquery/src/main/java/com/google/cloud/teleport/v2/utils/BigQueryAvroUtils.java

```diff
@@ -0,0 +1,525 @@
+/*
```

Comment:
If this was temporarily copied from Beam SDK to make some fixes, please make it clear in comments/javadoc.

Reply:
Done